run/test/specs/linked_device_avatar_color.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { bothPlatformsIt } from '../../types/sessionIt';
+import { USERNAME } from '../../types/testing';
+import { linkedDevice } from './utils/link_device';
+import { SupportedPlatformsType, closeApp, openAppTwoDevices } from './utils/open_app';
+import { compareColors, parseDataImage } from './utils/check_colour';
+import { UserSettings } from './locators/settings';
+
+bothPlatformsIt('Avatar color linked device', 'medium', avatarColorLinkedDevice);
+
+async function avatarColorLinkedDevice(platform: SupportedPlatformsType) {
+  const { device1, device2 } = await openAppTwoDevices(platform);
+  const userA = await linkedDevice(device1, device2, USERNAME.ALICE, platform);
+  // Get Alice's avatar color on device 1 and turn it into a hex value
+  const device1Avatar = await device1.waitForTextElementToBePresent(new UserSettings(device1));
+  const device1Base64 = await device1.getElementScreenshot(device1Avatar.ELEMENT);
+  const device1PixelColor = await parseDataImage(device1Base64);
+  // Get Alice's avatar color on the linked device and turn it into a hex value
+  const device2Avatar = await device2.waitForTextElementToBePresent(new UserSettings(device2));
+  const device2Base64 = await device2.getElementScreenshot(device2Avatar.ELEMENT);
+  const device2PixelColor = await parseDataImage(device2Base64);
+  // Color comparison of devices 1 and 2
+  const colorCompare = compareColors(device1PixelColor, device2PixelColor);
+  if (!colorCompare) {
```

Comment:
a bit of a weird name as it is a boolean. Maybe name it something that matches better, like `colorMatch` or `closeEnough`

"compare" in IT kind of sounds like it's returning -1, 0 or 1, depending on if the color is smaller, the same or bigger (whatever that'd mean for colors :joy: )

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#comparefn